pmpro-import-users-from-csv.php

```diff
@@ -203,6 +203,14 @@ function pmproiufcsv_is_iu_post_user_import($user_id)
 	if ( ! empty( $membership_timestamp ) ) {
 		$membership_timestamp = date( 'Y-m-d', strtotime($membership_timestamp, current_time( 'timestamp' ) ) );
 	}	
+
+	if ( ! empty( $membership_discount_code ) && empty( $membership_code_id ) )
+		$membership_code_id = $wpdb->get_var("SELECT id FROM $wpdb->pmpro_discount_codes WHERE `code` = '" . esc_sql($membership_discount_code) . "' LIMIT 1");		

```

Comment:
Let's use `$wpdb->prepare()` for any new queries we create, it ensures we follow coding best-practices for queries with WP and also helps avoid where WP dev docs says to avoid using `esc_sql()` in this way.

```suggestion
		$membership_code_id = $wpdb->get_var(
			$wpdb->prepare( "
				SELECT id
				FROM $wpdb->pmpro_discount_codes
				WHERE `code` = %s
				LIMIT 1
			", $membership_discount_code );
```